namespaces/live-1.cloud-platform.service.justice.gov.uk/crime-portal-mirror-gateway-preprod/resources/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "application" {
+  default = "court-case-service"
+}
+
+variable "namespace" {
+  default = "crime-portal-mirror-gateway-preprod"
+}
+
+variable "business-unit" {
+  description = "Area of the MOJ responsible for the service."
+  default     = "HMPPS"
+}
+
+variable "team_name" {
+  description = "The name of your development team"
+  default     = "probation-in-court"
+}
+
+variable "environment-name" {
+  description = "The type of environment you're deploying to."
```

Comment:
Ah, and here's where it's defined! Should this be dev or preprod?

Reply:
preprod. Next time we're in dev it might be worthwhile updating to use the var references